src/versuchung/archives.py

```diff
@@ -195,7 +196,7 @@ def before_experiment_run(self, parameter_type):
             Type.before_experiment_run(self, parameter_type)
 
     def __references(self, prefix_filter=None, regex_filter=None):
-        (lines, ret) = shell("git ls-remote %s 'refs/*'", self.__clone_url)
+        (lines, ret) = shell("git ls-remote %s 'refs/*'", self.__clone_url, stderr=PIPE)
```

Comment:
Wouldn't be `stderr=sys.stderr` be more useful than to create a pipe that is never read by somebody. Even worse, if the stderr-pipe is full, the process will block, or?